grid/src/Cabana_Grid_FastFourierTransform.hpp

```diff
@@ -129,7 +149,7 @@ class FastFourierTransformParams
       \brief Get MPI communication strategy.
       \return Using AllToAll or not.
     */
-    bool getAllToAll() const { return alltoall; }
```

Comment:
@jstewart28 @patrickb314 just wanted to check with you both since you're using Cabana+heffte. I don't think this will affect you, but this get method will break compatibility if I missed where you use it. I noticed you have a large switch/case block - does this help you?

Reply:
@streeve as long as `setAllToAll` still accepts `true` (it looks like it does) this won't break anything. The switch block allows us to dynamically set Alltoall parameters at runtime, so if anything, we can expand this switch block to cover the additional Alltoall parameters.